stylesheet.css

```diff
@@ -1,282 +1,61 @@
-    margin-top: -22px;
-    padding: 16px;
-    color: white;
-    font-weight: bold;
-    font-size: 18px;
-    transition: 0.6s ease;
-    border-radius: 0 3px 3px 0;
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-  }
-  
-  /* Position the "next button" to the right */
-  .next {
-    right: 0;
-    border-radius: 3px 0 0 3px;
-  }
-  
-  /* On hover, add a black background color with a little bit see-through */
-  .prev:hover,
-  .next:hover {
-    background-color: rgba(0, 0, 0, 0.8);
-  }
-  
-  /* Caption text */
-  .text {
-    color: #f2f2f2;
-    font-size: 15px;
-    padding: 8px 12px;
-    position: absolute;
-    bottom: 8px;
-    width: 100%;
-    text-align: center;
-  }
-  
-  /* Number text (1/3 etc) */
-  .numbertext {
-    color: #f2f2f2;
-    font-size: 12px;
-    padding: 8px 12px;
-    position: absolute;
-    top: 0;
-  }
-  
-  /* The dots/bullets/indicators */
-  .dot {
-    cursor: pointer;
-    height: 15px;
-    width: 15px;
-    margin: 0 2px;
-    background-color: #bbb;
-    border-radius: 50%;
-    display: inline-block;
-    transition: background-color 0.6s ease;
-  }
-  
-  .active,
-  .dot:hover {
-    background-color: #717171;
-  }
-  
-  /* Fading animation */
-  .fade {
-    -webkit-animation-name: fade;
-    animation-name: fade;
-    -webkit-animation-duration: 1.5s;
-    animation-duration: 1.5s;
-  }
-  
-  @-webkit-keyframes fade {
-    from {
-      opacity: 0.4;
-    }
-  
-    to {
-      opacity: 1;
-    }
-  }
-  
-  @keyframes fade {
-    from {
-      opacity: 0.4;
-    }
-  
-    to {
-      opacity: 1;
-    }
-  }
-  
-  .loginbg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .adminbg
-  {
-    background-color:pink;
-  }
-  .servicebg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .gallerybg
-  {
-    background-image: url("images/sleepy_kid.jpg");
-    background-size: cover;
-  }
-  
-  .contact Usbg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .girl_woods
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .finding_us
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .phone_photo
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .vintage_camera
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  form {
-    display: flex;
-    flex-direction: column;
-  }
-  
-  forn > span {
-    color: red;
-  }
-  
-  form > * {
-    padding-top: 15px;
-    padding-bottom: 15px;
-  }
-  
-  form > input {
-    padding: 15px;
-    border: none;
-    background-color: rgb(235, 235, 235);
-  }
-  
-  form > button {
-    padding: 15px;
-  }
-  
-  form > textarea {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-  }
-  
-  /*# sourceMappingURL=style.css.map */
\ No newline at end of file
+  background-color: #181818;
+  width: 100%;
+  height: 100vh;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  overflow-x: hidden;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  color:white;
+}
```

Comment:
To centre a single item on a page you can simply add `display: grid;` and `place-items: center;`.